scripts/issue/909/seed_badger_reth.py

```diff
@@ -0,0 +1,60 @@
+from pycoingecko import CoinGeckoAPI
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+SEED_PER_TOKEN_IN_DOLLAR = 1_000_000
+
+
+def main():
+    vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+    vault.init_balancer()
+    vault.init_aura()
+
+    # tokens and pool involved
+    badger = vault.contract(r.treasury_tokens.BADGER)
+    weth = vault.contract(r.treasury_tokens.WETH)
+    reth = vault.contract(r.treasury_tokens.RETH)
+    bpt_pool_seed = vault.contract(r.balancer.B_50_BADGER_50_RETH)
+    bpt_pool_swap = vault.contract(r.balancer.B_50_WETH_50_RETH)
+
+    # snap
+    vault.take_snapshot(tokens=[badger, weth])
+
+    # 1. calc needed amounts 1m each token
+    ids = ["badger-dao", "weth"]
+    prices = CoinGeckoAPI().get_price(ids, "usd")
+
+    badger_price = prices["badger-dao"]["usd"]
+    weth_price = prices["weth"]["usd"]
+
+    badger_mantissa = int((SEED_PER_TOKEN_IN_DOLLAR / badger_price) * 1e18)
+    weth_mantissa = int((SEED_PER_TOKEN_IN_DOLLAR / weth_price) * 1e18)
```

Comment:
```suggestion
    weth_mantissa = int((SEED_PER_TOKEN_IN_DOLLAR / weth_price) * 1e18 / 0.98)
```
would correct for the dusty deposit here

Reply:
sounds good, over-swapping to ensure that the dusty one is more precise